src/mono/mono/mini/mini-riscv.c

```diff
@@ -1703,12 +1702,14 @@ mono_arch_decompose_opts (MonoCompile *cfg, MonoInst *ins)
 
 	case OP_IMUL:
 	case OP_LMUL:
+	case OP_RMUL:
```

Comment:
sure，I will submit a pr of the progress so far. 
I think it may fix this problem.
And I plan to use monthly and half monthly pr to synchronize changes in the future

Reply:
have submit a pr https://github.com/dotnet/runtime/pull/85445
Now we are searching for when to push/pop lmf so that tey-catch can be executed correctly and mono can print the call stack correctly.